src/compute/src/server.rs

```diff
@@ -831,8 +831,23 @@ impl<'w, A: Allocate> Worker<'w, A> {
             // Remove all pending peeks.
             compute_state.pending_peeks.clear();
             // We compact away removed frontiers, and so only need to reset ids we continue to use.
-            for (_, frontier) in compute_state.reported_frontiers.iter_mut() {
-                *frontier = timely::progress::Antichain::from_elem(<_>::minimum());
+            // We must remember, though, to compensate what already was sent to logging sources.
+            for (id, frontier) in compute_state.reported_frontiers.iter_mut() {
+                let minimum = timely::progress::Antichain::from_elem(<_>::minimum());
+                if let Some(logger) = &compute_state.compute_logger {
+                    if let Some(time) = frontier.get(0) {
+                        use crate::logging::compute::ComputeEvent;
+                        logger.log(ComputeEvent::Frontier(*id, *time, -1));
+                        logger.log(ComputeEvent::Frontier(
+                            *id,
+                            *(minimum
+                                .get(0)
+                                .expect("minimum antichain should not be empty")),
+                            1,
```

Comment:
```suggestion
                        logger.log(ComputeEvent::Frontier(*id, Timestamp::minimum(), 1));
```
There is no need to unwrap an Antichain here in each iteration, when we can just supply the minimum timestamp directly.

Reply:
Cool, incorporated.